src/Tickets/Commerce/Hooks.php

```diff
@@ -701,4 +703,14 @@ public function register_event_compatibility_hooks() {
 
 		add_filter( 'wp_redirect', [ tribe( Compatibility\Events::class ), 'prevent_filter_redirect_canonical' ], 1, 2 );
 	}
+
+	/**
+	 * Includes the Assets to the checkout page.
+	 *
+	 * @since TBD
+	 */
+	public function include_assets_checkout_shortcode() {
```

Comment:
Should we maybe move this to the checkout shortcode class?

Reply:
It's just a line, but as we do it in other places, I think we may want to do the same.

Reply:
Good point moving.